pkg/terraform/diagnose.go

```diff
@@ -78,4 +78,9 @@ var conditions = []condition{{
 
 	reason:  "GCPComputeBackendTimeout",
 	message: `GCP is experiencing backend service interuptions, the compute instance failed to create in reasonable time.`,
+}, {
+	match: regexp.MustCompile(`Error: could not contact API: context deadline exceeded`),
+
+	reason:  "BaremetalIronicAPITimeout",
+	message: `Timed out connecting to Ironic API. Please make sure that Ironic API service is up and running.`,
```

Comment:
Was this committed by accident?

Reply:
Oh my god. Yes. I was working on three different branches. Thanks for the catch. Will update the commit.